Examples/Examples/FormExampleView.swift

```diff
@@ -41,15 +41,16 @@ struct FormExampleView: View {
                 }
                 .task(id: identifyScreenPoint) {
                     if let feature = await identifyFeature(with: mapViewProxy),
-                       let formDefinition = (feature.table?.layer as? FeatureLayer)?.featureFormDefinition {
-                        featureForm = FeatureForm(feature: feature, definition: formDefinition)
-                        formViewModel.startEditing(feature)
-                        isPresented = featureForm != nil
+                       let formDefinition = (feature.table?.layer as? FeatureLayer)?.featureFormDefinition,
+                       let featureForm = FeatureForm(feature: feature, definition: formDefinition) {
+                        self.featureForm = featureForm
+                        formViewModel.startEditing(feature, featureForm: featureForm)
                     }
+                    isPresented = featureForm != nil
                 }
                 .ignoresSafeArea(.keyboard)
-                
-                // Or present a FormView in a Floating Panel (provided via the Toolkit)
+            
+            // Or present a FormView in a Floating Panel (provided via the Toolkit)
```

Comment:
Indentation looks off but this will be resolved in #467 which I can merge after this PR is merged.